vir/simd.h

```diff
@@ -513,24 +513,26 @@ namespace vir::stdx
 
       // load constructor
       template <typename Flags>
+        constexpr
```

Comment:
I would suggest to put this part into its own commit.

Reply:
Yea..ah okay. :) This is a behavior enhancement...